Source/Engine/Platform/Base/WindowBase.cpp

```diff
@@ -439,6 +440,9 @@ void WindowBase::OnGotFocus()
         return;
     _focused = true;
 
+    if (RenderTask && _settings.WindowFPSWhenNotFocused > ZeroTolerance)
```

Comment:
Is `_settings.WindowFPSWhenNotFocused` an integer? If so, just writing `_settings.WindowFPSWhenNotFocused > 0` should do the trick.